### PR TITLE
More hs2 content

### PR DESCRIPTION
**Pull request by wojtekkedzior**
Adding the second large batch of changes for the new Hotel Service 2 API.  All changes are only related to the content.  This should be the last major update.  From here on in, we should be making minor changes.

### CODE REVIEW

File: api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown [hunk 1]
```diff
@@ -3,34 +3,97 @@ title: Authentication and Errors
 layout: reference
 ---
 
+**rename page to Error handling**
```
Comment (bhague1281) [inline api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown]: This should be removed and page should be renamed. You'll want to do the following:

1. Add a title to the markdown file. E.g.

```yml
---
layout: reference
title: Error Handling
---
```

2. Update the title/url in the sidebar here: https://github.com/concur/developer.concur.com/blob/preview/_data/sidebars/api-reference.yml#L113

File: api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown [hunk 2]
```diff
@@ -3,34 +3,97 @@ title: Authentication and Errors
...
 
-Authentication
 
-Concur will send the username and password in both the HTTP header and the SOAP header.  The HTTP header follows the best practices of username:password which is Base64 encoded.   
+Errors
```
Comment (bhague1281) [inline api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown]: This should be a level 2 header

File: api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown (hunk 1 above)
Comment (wojtekkedzior) [reply api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown]: Done. Thanks for picking that up.

File: api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown (hunk 2 above)
Comment (wojtekkedzior) [reply api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown]: Also done.

**Pull request merged**